### PR TITLE
konflux: fix hermetic build argument

### PR DESCRIPTION
**Pull request by raptorsun**
## Description

fix the hermetic argument to trigger hermetic build for build-container

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by tisnik — COMMENTED** [review]
According to https://konflux-ci.dev/docs/how-tos/configuring/hermetic-builds/ it is the correct parameter name, indeed!

**Comment by raptorsun** [comment]
/hold
remove the test commit before merging

File: Containerfile [hunk 1]
```diff
@@ -23,18 +21,16 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
...
 COPY --from=lightspeed-rag-content /rag/vector_db/ocp_product_docs ./vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-content /rag/embeddings_model ./embeddings_model
 
 # Add explicit files and directories
 # (avoid accidental inclusion of local directories or env files or credentials)
-COPY pyproject.toml pdm.lock runner.py ./
-RUN pip3.11 install --no-cache-dir --upgrade pip pdm==2.18.1 \
-    && pdm config python.use_venv false \
-    && pdm sync --global --prod -p ${APP_ROOT}
+COPY pyproject.toml pdm.lock runner.py requirements.txt ./
```
Comment (syedriko) [inline Containerfile:31]: Do we still need to copy pyproject.toml and pdm.lock?

Comment (raptorsun) [reply Containerfile:31]: no, lets remove them

**Comment by raptorsun** [comment]
/close 
in favor of https://github.com/openshift/lightspeed-service/pull/1731

**Pull request closed without merging**